### PR TITLE
added trello action

### PR DESCRIPTION
**Pull request by ldrr**
```
+-------------------------------------------------------------------------------+
|                                    trello                                     |
+-------------------------------------------------------------------------------+
| Sets the :TRELLO_WHAT_TO_TEST from titles from a trello board & moves tickets |
|                                                                               |
| Created by ldrr                                                               |
+-------------------------------------------------------------------------------+

+--------------+---------------------------+------------------------+
|                              trello                               |
+--------------+---------------------------+------------------------+
| Key          | Description               | Env Var                |
+--------------+---------------------------+------------------------+
| public_key   | Public API Key for Trello | FL_TRELLO_PUBLIC_KEY   |
| member_token | Member Token for Trello   | FL_TRELLO_MEMBER_TOKEN |
| board_source | ID of trello source board | FL_TRELLO_BOARD_SOURCE |
| board_target | ID of trello target board | TRELLO_BOARD_TARGET    |
+--------------+---------------------------+------------------------+

+---------------------+---------------------------------+
|                        trello                         |
+---------------------+---------------------------------+
| Key                 | Description                     |
+---------------------+---------------------------------+
| TRELLO_WHAT_TO_TEST | List of all done-items in Board |
+---------------------+---------------------------------+
```

### CODE REVIEW

**Comment by KrauseFx** [comment]
Sorry for the delay from my side. I'll test the new action the next few days and report back :+1:

File: lib/fastlane/actions/trello.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        folder = '.' #subfolder folder is the default folder
+
+        command_prefix = [
+            'cd',
+            File.expand_path(folder).shellescape,
+            '&&'
+        ].join(' ')
+
+        begin
+#          Actions.lane_context[SharedValues::WHAT_TO_TEST] = ""
```
Comment (KrauseFx) [inline lib/fastlane/actions/trello.rb]: Comment should be removed?

File: lib/fastlane/actions/trello.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          current_version = `#{command_prefix} agvtool what-marketing-version -terse1`.split("\n").last || ''
+          build_number = `#{command_prefix} agvtool what-version`.split("\n").last.to_i
+          title = "Version #{current_version} (#{build_number})"
+
+          board = Trello::Board.find(params[:board_source]) 
+  
+          list = board.lists.select{ |l| l.name == 'Done'}.first
```
Comment (KrauseFx) [inline lib/fastlane/actions/trello.rb]: Maybe the documentation should mention that the column needs to be named `Done`?

**Comment by KrauseFx** [comment]
The integration looks really nice but needs a better documentation on how to set it up and how to use it.
You can implement `def self.details` in the `trello` action to provide further details with information like the `Done` column and if the integration modifies any cards.

Additionally, please add the new Trello action to the [actions documentation](https://github.com/KrauseFx/fastlane/blob/master/docs/Actions.md) as well.

Thanks again for your contribution :+1:

**Comment by ldrr** [comment]
As soon as I will find some time I'll add the documentation. But in the meanwhile: sorry for the delay 😵

**Comment by KrauseFx** [comment]
No problem, I'll close this pull request for now, please feel free to submit a new one :+1:

**Pull request closed without merging**